<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CharlyRobot</class>
 <widget class="QMainWindow" name="CharlyRobot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1195</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CharlyRobot</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="GroupeManu">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>190</y>
      <width>501</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>                                                 Gestion Manuelle de CHARLYROBOT :</string>
    </property>
    <widget class="QLabel" name="TexteY_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>80</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Y :</string>
     </property>
    </widget>
    <widget class="QLabel" name="TexteX_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X :</string>
     </property>
    </widget>
    <widget class="QLabel" name="TexteZ_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Z :</string>
     </property>
    </widget>
    <widget class="QSlider" name="DeplacementZ_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>21</width>
       <height>251</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="maximum">
      <number>95</number>
     </property>
     <property name="tracking">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedAppearance">
      <bool>true</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QSlider" name="DeplacementX_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>401</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="maximum">
      <number>280</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="DeplacementY_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>401</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximum">
      <number>160</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="ButtonPositionner">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>200</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#ButtonPositionner{
background: rgb(52, 152, 219);
border-radius:5px;
 }
#ButtonPositionner::hover { 
background : rgb(21, 67, 96 );
color:white;
}
</string>
     </property>
     <property name="text">
      <string>Positionner</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="Valeur_Z_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#fff;
color: black;</string>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="Valeur_X_2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>40</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>New Century Schoolbook</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#fff;
color: black;</string>
     </property>
     <property name="maximum">
      <number>280</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="Valeur_Y_2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>80</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#fff;
color: black;</string>
     </property>
     <property name="maximum">
      <number>300</number>
     </property>
    </widget>
    <widget class="QSlider" name="Changement_Vitesse">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>401</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1000</number>
     </property>
     <property name="maximum">
      <number>2000</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="Texte_Manuelle_Vitesse">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>130</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Vitesse :</string>
     </property>
    </widget>
    <widget class="QLabel" name="mm">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>40</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>New Century Schoolbook</family>
      </font>
     </property>
     <property name="text">
      <string>mm</string>
     </property>
    </widget>
    <widget class="QLabel" name="mm_2">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>80</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>New Century Schoolbook</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>mm</string>
     </property>
    </widget>
    <widget class="QLabel" name="mm_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>mm</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="Valeur_Manuelle_Vitesse">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>120</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#fff;
color: black;</string>
     </property>
     <property name="minimum">
      <number>1000</number>
     </property>
     <property name="maximum">
      <number>2000</number>
     </property>
    </widget>
    <widget class="QPushButton" name="Valider_Origine">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>200</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#Valider_Origine{
background: rgb(52, 152, 219);
border-radius:5px;
 }
#Valider_Origine::hover { 
background : rgb(21, 67, 96 );
color:white;
}
</string>
     </property>
     <property name="text">
      <string>Validation Point D'origine</string>
     </property>
    </widget>
    <widget class="QPushButton" name="reset">
     <property name="geometry">
      <rect>
       <x>184</x>
       <y>200</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#reset{
background: rgb(52, 152, 219);
border-radius:5px;
 }
#reset::hover { 
background : rgb(21, 67, 96 );
color:white;
}
</string>
     </property>
     <property name="text">
      <string>Réinitialiser</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="GroupeReel">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>20</y>
      <width>311</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>New Century Schoolbook</family>
     </font>
    </property>
    <property name="title">
     <string>         CHARLYROBOT : Coordonnées en temps réel. </string>
    </property>
    <widget class="QLabel" name="TexteReelX_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>X :</string>
     </property>
    </widget>
    <widget class="QLabel" name="TexteReelY_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Y :</string>
     </property>
    </widget>
    <widget class="QLabel" name="TexteReelZ_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Z :</string>
     </property>
    </widget>
    <widget class="QLabel" name="ValeurReelX_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>000</string>
     </property>
    </widget>
    <widget class="QLabel" name="ValeurReelY_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>000</string>
     </property>
    </widget>
    <widget class="QLabel" name="ValeurReelZ_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>000</string>
     </property>
    </widget>
    <widget class="QLabel" name="Valeur_Reel_Vitesse">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>000</string>
     </property>
    </widget>
    <widget class="QLabel" name="Texte_Reel_Vitesse">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Vitesse de déplacement :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>mm</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>mm</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>70</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>mm</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>tr/s</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="Demarrer">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>500</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#Demarrer{
background: rgb(52, 152, 219);
border-radius:5px;
	font: 87 9pt &quot;Arial Black&quot;;
 }
#Demarrer::hover { 
background : rgb(21, 67, 96 );
color:white;
}
</string>
    </property>
    <property name="text">
     <string>Démarrer </string>
    </property>
   </widget>
   <widget class="QComboBox" name="listcom">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>530</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: grey;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>530</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>New Century Schoolbook</family>
     </font>
    </property>
    <property name="text">
     <string>Port COM: </string>
    </property>
   </widget>
   <widget class="QLabel" name="Texte_FichierDXF">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Arial;
font-size:12px;</string>
    </property>
    <property name="text">
     <string>Fichier DXF ouvert avec succès :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Boutton_Visuel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
#Boutton_Visuel{
background: rgb(52, 152, 219);
border-radius:5px;
 }
#Boutton_Visuel::hover { 
background : rgb(21, 67, 96 );
color:white;
}

</string>
    </property>
    <property name="text">
     <string>Visuel De La Gravure</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>200</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>New Century Schoolbook</family>
     </font>
    </property>
    <property name="text">
     <string>Suivi de la Fraise : </string>
    </property>
   </widget>
   <widget class="QPushButton" name="Deconnexion">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>530</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
#Deconnexion{
background: rgb(52, 152, 219);
border-radius:5px;
	font: 9pt &quot;Arial&quot;;
 }


#Deconnexion::hover{ 
background : rgb(21, 67, 96 );
color:white;
}
</string>
    </property>
    <property name="text">
     <string>Déconnexion</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Connexion">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
#Connexion{
font: 9pt &quot;Arial&quot;;
background: rgb(52, 152, 219);
border-radius:5px;
 }
#Connexion::hover { 
background : rgb(21, 67, 96 );
color:white;
}

</string>
    </property>
    <property name="text">
     <string>Connexion</string>
    </property>
   </widget>
   <widget class="QLabel" name="state">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>532</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>20</y>
      <width>181</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Point D'origine</string>
    </property>
    <widget class="QLabel" name="X_Origine">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X : </string>
     </property>
    </widget>
    <widget class="QLabel" name="Y_Origine">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Y :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Z :</string>
     </property>
    </widget>
    <widget class="QLabel" name="Valeur_X_Origine">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>000</string>
     </property>
    </widget>
    <widget class="QLabel" name="Valeur_Y_Origine">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>47</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>000</string>
     </property>
    </widget>
    <widget class="QLabel" name="Valeur_Z_Origine">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>000</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="effme">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>490</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Efface mémoire</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>512</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_police">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Saisie police :</string>
    </property>
   </widget>
   <widget class="QComboBox" name="c_police">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Arial</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Futura</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Blackadder</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tmes New Roman</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Verdana</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="b_police">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Valider</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1195</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#menuBar{
 background:rgb(59, 64, 61  );	
color : white;
   }
#menuBar::item:selected { 
background : rgb(77, 86, 86  );;
color:white;
}

</string>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="styleSheet">
     <string notr="true">#menuFichier{
 background:rgb(59, 64, 61  );	
color : white;
border-bottom:2px;
   }
#menuFichier::item:selected { 
background : rgb(77, 86, 86  );;
color:white;
}

</string>
    </property>
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="Ouvrir"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="styleSheet">
     <string notr="true">#menuOptions{
 background:rgb(59, 64, 61  );	
color : white;
border-bottom:2px;
   }
#menuOptions::item:selected { 
background : rgb(77, 86, 86  );;
color:white;
}
</string>
    </property>
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="Plaque"/>
    <addaction name="Fraise"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="Ouvrir">
   <property name="text">
    <string>Ouvrir</string>
   </property>
  </action>
  <action name="Plaque">
   <property name="text">
    <string>Reglage Plaque</string>
   </property>
  </action>
  <action name="Fraise">
   <property name="text">
    <string>Reglage Fraise</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>DeplacementZ_2</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>Valeur_Z_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>657</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>713</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DeplacementX_2</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>Valeur_X_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>907</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>907</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DeplacementY_2</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>Valeur_Y_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>851</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>902</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Valeur_Z_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DeplacementZ_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>723</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>663</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Valeur_X_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DeplacementX_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>910</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>958</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Valeur_Y_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DeplacementY_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>921</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>969</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DeplacementZ_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Valeur_Z_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>669</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>733</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DeplacementX_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Valeur_X_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>850</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>906</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DeplacementY_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Valeur_Y_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>830</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>893</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Changement_Vitesse</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>Valeur_Manuelle_Vitesse</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>943</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>900</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Valeur_Manuelle_Vitesse</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Changement_Vitesse</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>896</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>978</x>
     <y>369</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Changement_Vitesse</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Valeur_Manuelle_Vitesse</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>997</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>923</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
